front/pages/api/v1/w/[wId]/vaults/[vId]/apps/[aId]/runs/[runId]/index.ts

```diff
@@ -29,34 +29,24 @@ async function handler(
   res: NextApiResponse<WithAPIErrorResponse<GetRunResponseBody>>,
   auth: Authenticator
 ): Promise<void> {
-  const owner = auth.workspace();
-  if (!owner) {
-    return apiError(req, res, {
-      status_code: 404,
-      api_error: {
-        type: "workspace_not_found",
-        message: "The workspace you're trying to access was not found",
-      },
-    });
-  }
-
-  let vaultId = req.query.vId;
+  const owner = auth.getNonNullableWorkspace();
```

Comment:
I think we should never use non nullable workspace in handlers. This should be 404 not 500

Reply:
Well here it is unexpected as we already check for this in the authentication wrapper

Reply:
We standardized on handling this in warppers and using nonnullable in most endpoints/handlers